webapp/channels/src/components/admin_console/access_control/editors/table_editor/table_editor.tsx

```diff
@@ -89,204 +75,279 @@ const parseExpression = async (expr: string): Promise<TableRow[]> => {
-                                        updateValues={(values) => updateRowValues(index, values)}
+                                        updateValues={(values: string[]) => updateRowValues(index, values)}
+                                        options={row.attribute ? userAttributes.find((attr) => attr.name === row.attribute)?.attrs?.options || [] : []}
                                     />
-                                </div>
-                                <div className='table-editor__cell-actions'>
+                                </td>
+                                <td className='table-editor__cell-actions'>
                                     <button
+                                        type='button'
                                         className='table-editor__row-remove'
                                         onClick={() => removeRow(index)}
                                         disabled={disabled}
                                         aria-label={formatMessage({id: 'admin.access_control.table_editor.remove_row', defaultMessage: 'Remove row'})}
                                     >
                                         <i className='icon icon-trash-can-outline'/>
                                     </button>
-                                </div>
-                            </div>
+                                </td>
+                            </tr>
                         ))
                     )}
-                </div>
-                <div className='table-editor__add-button-container'>
-                    <AddAttributeButton
-                        onClick={addRow}
-                        disabled={disabled || getAvailableAttributes().length === 0}
-                    />
-                </div>
-            </div>
+                </tbody>
+                <tfoot>
+                    <tr>
+                        <td
+                            colSpan={4}
+                            className='table-editor__add-button-container'
+                        >
+                            <AddAttributeButton
+                                onClick={addRow}
+                                disabled={disabled || userAttributes.length === 0}
+                            />
+                        </td>
+                    </tr>
+                </tfoot>
+            </table>
 
             <div className='table-editor__actions-row'>
                 <HelpText
-                    message={'Each row is a single condition that must be met for a user to comply with the policy. All rules are combined with logical AND operator (`&&`).'}
+                    message={formatMessage({
+                        id: 'admin.access_control.table_editor.help_text',
+                        defaultMessage: 'Each row is a single condition that must be met for a user to comply with the policy. All rules are combined with logical AND operator (`&&`).',
```

Comment:
> This text also needs to be translatable, but I think we should also change this to remove the Markdown from it as well

https://github.com/mattermost/mattermost/pull/30785#discussion_r2089076534 comment from @hmhealey 

Updated text to use translation string